showcase/src/app/home/_home.component.scss

```diff
@@ -0,0 +1,66 @@
+//custom title for the header. Keep in mind that we define the font-size and color here because it is an exceptional case.
+//Should this style be reused elsewhere, it would be better to create a new style in stark-ui
+.about-title {
+  font-size: 30px;
+  margin-left: 15px;
+  color: mat-color($grey-palette, 100);
+}
+
+//custom title for the collapsible
+mat-panel-title {
+  font-size: 25px;
```

Comment:
We should avoid to set this type of font here. If we want to change the size of the text, either we can use tags (h1, h2 etc...), either we need to set it in the collapsible component.
I will go for the first option :-)